Makefile

```diff
@@ -62,6 +70,36 @@ code-gen: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 generate-clients:
 	GO=GO111MODULE=on GOTOOLCHAIN=go1.23.4 GOFLAGS=-mod=readonly hack/update-codegen.sh
 
+.PHONY: bundle-build
+bundle-build: operator-sdk regen-crd manifests
+	${OPERATOR_SDK} generate bundle --input-dir deploy/ --version ${OPERATOR_VERSION}
+
+.PHONY: deploy-ocp
+deploy-ocp: 
+	hack/deploy-ocp.sh ${OPERATOR_IMAGE} ${KUEUE_IMAGE}
+
+.PHONY: undeploy-ocp
+undeploy-ocp:
+	hack/undeploy-ocp.sh
+
+# Below targets require you to login to your registry
+.PHONY: podman-operator-build
+podman-operator-build:
```

Comment:
These can just be operator-build, operator-push, etc now.

Reply:
yep, good idea.